code/modules/spacepods/equipment.dm

```diff
@@ -8,7 +8,9 @@
 		to_chat(usr, "<span class='warning'>Missing equipment or weapons.</span>")
 		my_atom.verbs -= text2path("[type]/proc/fire_weapons")
 		return
-	my_atom.battery.use(shot_cost)
+	if(0 == my_atom.battery.use(shot_cost))
```

Comment:
You shouldn't have merge commits in a freshly submitted PR. Please follow these instructions here: https://github.com/ParadiseSS13/Paradise/pull/5933#issuecomment-264594234

Secondly, the way this is checked for isn't optimal at all. 

```
if(!my_atom.battery.use(shot_cost))
	to_chat(usr, "<span class='warning'>Insufficient charge to fire the weapons</span>")
	return
```

Would be better.